docs/study_viewer.md

```diff
@@ -8,7 +8,7 @@ Example configuration:
         "dataType": "dataset",
         "title": "Datasets",           // page title
         "titleField": "name",         // row title
-        "listItemConfig": {  // required
+        "listItemConfig": {
```

Comment:
We probably should fix the code not this doc, `listItemConfig` was intended to be a madatory section 🤦

Reply:
Is it not anymore? This change only makes it clear what is required and what is not. So all fields not labeled as optional are required. It is confusing to have some labelled as required and some labelled as optional and some not labelled as either.

Reply:
I misread your comment, sorry